spec/lib/js_locale_helper_spec.rb

```diff
@@ -276,9 +278,11 @@ def clear_cache!
         value: "SHOULD_NOT_SHOW",
         status: "deprecated",
       )
+    end
 
-      expect(described_class.output_client_overrides("en").include? "SHOULD_SHOW").to eq(true)
-      expect(described_class.output_client_overrides("en").include? "SHOULD_NOT_SHOW").to eq(false)
+    it "does not output deprecated translation overrides" do
+      expect(client_overrides).to include("SHOULD_SHOW")
```

Comment:
This is more RSpec idiomatic to use dedicated matchers. It reads a bit better too :grin: